### PR TITLE
kerberos failures aren't clearing though collections uninterrupted

### PR DESCRIPTION
**Pull request by alexvashchyshyn**

### CODE REVIEW

File: ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py [hunk 1]
```diff
@@ -264,9 +264,9 @@ def onSuccess(self, results, config):
         ds0 = config.datasources[0]
 
         eventlog = ds0.params['eventlog']
-
+
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py]: remove whitespace, pep8

File: ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py [hunk 2]
```diff
@@ -264,9 +264,9 @@ def onSuccess(self, results, config):
         ds0 = config.datasources[0]
 
         eventlog = ds0.params['eventlog']
-
+         
         def _makeEvent(evt):
-            ds = ds0.params
+             ds = ds0.params
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py]: remove whitespace, this causes an IndentationError

File: ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py [hunk 3]
```diff
@@ -357,6 +357,7 @@ def _makeEvent(evt):
             'severity': ZenEventClasses.Clear,
             'eventKey': 'WindowsEventCollection: {}'.format(ds0.params.get('eventid', '')),
             'eventClassKey': 'WindowsEventLogSuccess',
+
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py]: remove newline

File: ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py [hunk 4]
```diff
@@ -368,7 +369,7 @@ def _makeEvent(evt):
                 'eventClass': '/Status/Winrm',
                 'eventClassKey': 'WindowsEventLogSuccess',
             })
-
+
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py]: remove whitespace

File: ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py [hunk 5]
```diff
@@ -398,7 +400,7 @@ def onError(self, result, config):
                     'message': msg,
                     'device': config.id
                 })
-        return data
+        return data
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py]: remove whitespace

File: ZenPacks/zenoss/Microsoft/Windows/datasources/PerfmonDataSource.py [hunk 6]
```diff
@@ -632,7 +636,7 @@ def onReceive(self, result):
         # Continue to receive if MaxSamples value has not been reached yet.
         elif self.collected_samples < self.max_samples and results:
             self.receive()
-
+
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/PerfmonDataSource.py]: remove whitespace

File: ZenPacks/zenoss/Microsoft/Windows/datasources/PortCheckDataSource.py [hunk 7]
```diff
@@ -168,6 +168,7 @@ def onSuccess(self, results, config):
             # no ports listening
             pass
         # send clear events
+
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/PortCheckDataSource.py]: remove newline

File: ZenPacks/zenoss/Microsoft/Windows/utils.py [hunk 8]
```diff
@@ -12,6 +12,7 @@
 """
 
 import json
+from Products.ZenEvents import ZenEventClasses
```
Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/utils.py]: see line 11 above.  we can't import ZenEventClasses

**Comment by dbouchillon** [comment]
Alex, you need to fix the failing unit tests.  Also, this branch needs to be based off of the hotfix/2.9.1 branch and make your pull request so that you are asking to merge it into the hotfix/2.9.1 branch...not the develop branch

**Comment by alexvashchyshyn** [comment]
just didn't push the last commit. reopening the pull request on hotfix/2.9.1 with all correct requirements

**Pull request closed without merging**